data/campaigns/Eastern_Invasion/scenarios/10_Lake_Vrug.cfg

```diff
@@ -29,7 +29,8 @@
         controller=human
         team_name=good
         user_team_name=_"Wesnothians"
-        shroud=yes
+        shroud=no
+        fog=yes
```

Comment:
@nemaara, you've rebalanced the scenario after this PR was opened, and I see you've given the player many more turns to explore.

Does changing the shroud to fog still make sense?

Reply:
Ya, fog still seems nice. No fog is pretty nasty in this scenario.